modules/sagemaker/sagemaker-templates-service-catalog/templates/finetune_llm_evaluation/pipeline_constructs/build_pipeline_construct.py

```diff
@@ -228,6 +234,16 @@ def __init__(
                 environment_variables={
                     "SAGEMAKER_PROJECT_NAME": codebuild.BuildEnvironmentVariable(value=project_name),
                     "SAGEMAKER_PROJECT_ID": codebuild.BuildEnvironmentVariable(value=project_id),
+                    **(
```

Comment:
Can we just pass through the value here and handle None/empty string in downstream code?

Reply:
Yeah, downstream code was already setup to handle it so have removed this in new commit

Reply:
Thank you!